packages/schema/src/tag-names.ts

```diff
@@ -1,7 +1,7 @@
 export enum TagEnum {
```

Comment:
Ich denke für den Moment passt hier alles. ✔

Ich würde nur gerne mal die Frage in den Raum stellen, ob der Enum nicht irgendwann einen klareren Namen verdient hätte. "TagEnum" in Schema ist ja erst mal sehr generisch. Hier würde ich von Namen her wohl alle vorhandenen Tag Names erwarten.

Tatsächlich kann ich gerade selber nicht sicher sagen, wie die Auswahl hier zustanden kommt: Ich glaube es sind jetzt alle "öffentlichen" Komponenten, die sich auch stylen lassen? Wäre es gut, wenn sich diese Info auch im Namen des Enum wiederfinden würde?